rampwf/tests/kits/titanic_no_test/titanic_no_test_starting_kit.ipynb

```diff
@@ -0,0 +1,1337 @@
+{
```

Comment:
no need to add the notebook here I think

Reply:
True, but doesn't the test need it?